doc/sphinx-guides/source/installation/config.rst

```diff
@@ -2068,6 +2068,8 @@ https://demo.dataverse.org/sitemap.xml is the sitemap URL for the Dataverse Proj
 
 Once the sitemap has been generated and placed in the domain docroot directory, it will become available to the outside callers at <YOUR_SITE_URL>/sitemap/sitemap.xml; it will also be accessible at <YOUR_SITE_URL>/sitemap.xml (via a *pretty-faces* rewrite rule). Some search engines will be able to find it at this default location. Some, **including Google**, need to be **specifically instructed** to retrieve it.
 
+According to `Sitemaps.org protocol <https://www.sitemaps.org/protocol.html#index>`_, a sitemap file must have no more than 50,000 URLs and must be no larger than 50MiB. In this case, you have to create sitemap index file witch references multiples sitemap files called ``sitemap_index.xml``, instead of ``sitemap.xml``. Be aware in previous steps to use the correct file name corresponding to your installation.
```

Comment:
```suggestion
According to `Sitemaps.org protocol <https://www.sitemaps.org/protocol.html#index>`_, a sitemap file must have no more than 50,000 URLs and must be no larger than 50MiB. In this case, the protocol instructs you to create a sitemap index file called ``sitemap_index.xml`` (instead of ``sitemap.xml``), which references multiples sitemap files. Be aware in previous steps to use the correct file name corresponding to your situation.
```

I'm suggesting "situation" but I'm not sure that's right. We're asking the sysadmin to be conscious of if they are above or below 50,000 URLs? Is this easy to check? Can we just always use the style that allows for over 50,000 URLs so the (poor) sysadmin doesn't have to think about it? 😅

Reply:
Thanks @pdurbin for your review :pray:

Do you think the following addition will help sysadmins ?

_As a URL corresponds to a published collection or dataset, it's possible to determine which file to use based on the sum of these elements._

This reminds me to mention that I didn't add datafile support in the sitemap. I considered that this could be done in another issue. But I'm open to other suggestions :)

Reply:
I just left a comment under the release note - a quick similar explanation would help here as well, IMO. Just spelling out that a single `sitemap.xml` is being replaced by the sitemap index **which in turn lists the multiple partial sitemap files, `sitemap_1.xml`, `sitemap_2.xml` ... etc.**
And maybe also a word or two explaining that starting with this version of Dataverse this will be done automatically, depending on how many public datasets and collections are hosted in a given instance. 

Seeing how this has been slightly confusing to a reviewer, adding these extra words may help a few remote admins understand what's going on.

Reply:
And no, I wouldn't worry about datafiles for now. It's been our policy to discourage googlebot from crawling the files, and nobody has yet requested to change it.